### PR TITLE
Run make-gen

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Review by shamsher31 — APPROVED** [review]

File: manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ metadata:
       The Maistra Operator enables you to install, configure, and manage an instance of Maistra service mesh. Maistra is based on the open source Istio project.
 
     containerImage: maistra/istio-ubi8-operator:2.1.0
-    createdAt: 2021-06-10T10:42:28IST
+    createdAt: 2021-07-07T12:02:04CEST
```
Comment (jwendell) [inline manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml]: Rob always ask to revert these changes - I forgot why. It doesn't hurt to me though.

File: manifests-servicemesh/2.1.0/servicemeshoperator.v2.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ metadata:
       The OpenShift Service Mesh Operator enables you to install, configure, and manage an instance of Red Hat OpenShift Service Mesh. OpenShift Service Mesh is based on the open source Istio project.
 
     containerImage: registry.redhat.io/openshift-service-mesh/istio-rhel8-operator:2.1.0
-    createdAt: 2021-06-10T10:42:30IST
+    createdAt: 2021-07-07T12:02:08CEST
```
Comment (jwendell) [inline manifests-servicemesh/2.1.0/servicemeshoperator.v2.1.0.clusterserviceversion.yaml]: ditto

File: manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (luksa) [reply manifests-maistra/2.1.0/maistraoperator.v2.1.0.clusterserviceversion.yaml]: Yeah, I typically revert them, too. Forgot to do it this time, but I've done it now.

File: manifests-servicemesh/2.1.0/servicemeshoperator.v2.1.0.clusterserviceversion.yaml (hunk 2 above)
Comment (luksa) [reply manifests-servicemesh/2.1.0/servicemeshoperator.v2.1.0.clusterserviceversion.yaml]: fixed.

**Review by jwendell — APPROVED** [review]
